ui/src/dashboards/actions/ranges.ts

```diff
@@ -91,6 +91,18 @@ export const updateQueryParams = (updatedQueryParams: object): RouterAction => {
   return replace(newLocation)
 }
 
+export const updateQueryVars = varsObj => {
+  const urlVars = qs.parse(window.location.search, {ignoreQueryPrefix: true})
+  const vars = {
+    ...(urlVars.vars || {}),
```

Comment:
I would rather see a get here. but it's a nit

Reply:
I BET YOU WOULD